log/sudoers_ts.ksy

```diff
@@ -0,0 +1,114 @@
+meta:
+  id: sudoers_ts
+  title: Sudoers Time Stamp file
+  license: CC0-1.0
+  endian: le
```

Comment:
As per original sudo implementation, it seems to be machine-dependent, not truly little endian?

Reply:
Indeed. What is the recommended way to define a machine-dependent endianness? Removing this line, or something else?

Reply:
You can use `version` of the first record to determine endianness. Then there is a way to set endiannes in `meta` using  `switch-on`-`cases`.